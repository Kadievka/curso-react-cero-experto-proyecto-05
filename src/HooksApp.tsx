import React from 'react'

const HooksApp = () => {
  return (
    <div data-testid="hooks-app-container-element" >HooksApp</div>
  )
}

export default HooksApp