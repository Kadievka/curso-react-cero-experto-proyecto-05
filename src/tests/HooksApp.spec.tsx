import React from 'react';
import {
  cleanup,
  render,
  screen,
} from "@testing-library/react";
import HooksApp from "../HooksApp";

describe('HooksApp', () => {

  beforeEach(()=>{
    cleanup();
  });

  it('should show the defined categories', () => {

    const wrapper = render(<HooksApp />);

    expect(wrapper.asFragment()).toMatchSnapshot();

    const container = screen.getByTestId('hooks-app-container-element');

    expect(container.innerHTML.trim()).toBe("HooksApp");

  });

});