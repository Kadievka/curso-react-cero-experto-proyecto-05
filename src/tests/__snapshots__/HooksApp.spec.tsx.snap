// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HooksApp should show the defined categories 1`] = `
<DocumentFragment>
  <div
    data-testid="hooks-app-container-element"
  >
    HooksApp
  </div>
</DocumentFragment>
`;
